Cargo.toml

```diff
@@ -25,6 +25,7 @@ default = ["rocksdb/snappy", "rocksdb/lz4", "rocksdb/zstd", "rocksdb/zlib", "roc
 base64 = "0.10"
 bincode = "1.0"
 bitcoin = { version = "0.26.0", features = ["use-serde"] }
+cbor = "0.4.1"
```

Comment:
Unfortunately this crate isn't maintained, you should switch to `serde_cbor`, the API should be similar.  See note in the repo: https://github.com/BurntSushi/rust-cbor